### DISCUSSION TITLE
Autoplay stops on clicking thumbnail slide

### DISCUSSION THREAD

**Discussion by bikrameb** [nolimits4web/swiper#6428] [Q&A]
Auto play  stop working  when the thumbnails  slide is changed  by clicking the slide  

Code:  
    var swiperLogo = new Swiper(".mySwiper", {
      loop: true,
      spaceBetween: 30,
      slidesPerView: 1,
      watchOverflow: true,
      allowTouchMove: false,
      noSwiping:false,
      breakpoints: {
        640: {
          slidesPerView: 3,
        },
        768: {
          slidesPerView: 4,
        },
        1024: {
          slidesPerView: 6,
        },
      },
      pagination: {
        el: ".swiper-pagination",
        clickable:true,
      },
      autoplay: {
        delay: 2000,
        autoplayDisableOnInteraction:false,
        pauseOnMouseEnter:false,
      },
    });
    var swiperText = new Swiper(".mySwiper2", {
      loop: true,
      spaceBetween: 30,
      autoHeight:true,
      navigation: {
        nextEl: ".swiper-button-next",
        prevEl: ".swiper-button-prev",
      },
      thumbs: {
        swiper: swiperLogo,
      },
      autoplay: {
        delay: 2000,
        autoplayDisableOnInteraction:false,
        pauseOnMouseEnter:false,
      },
    });

**Comment by haleyngonadi** [comment]
@bikrameb Did you figure this one out? Having the same issue.
